--- /etc/monit/monitrc	2017-01-11 14:48:27.000000000 +0100
+++ /etc/monit/monitrc	2019-01-27 12:25:00.000000000 +0100
@@ -17,5 +17,6 @@
 ## Start Monit in the background (run as a daemon):
 #
-  set daemon 120            # check services at 2-minute intervals
+# set daemon 120            # check services at 2-minute intervals
+  set daemon 30
 #   with start delay 240    # optional: delay the first check by 4-minutes (by
 #                           # default Monit check immediately after Monit start)
