--- /etc/ssh/sshd_config	2016-08-04 08:00:00.000000000 +0200
+++ /etc/ssh/sshd_config	2018-12-31 15:37:34.000000000 +0100
@@ -26,5 +26,5 @@
 # Authentication:
 LoginGraceTime 120
-PermitRootLogin yes
+PermitRootLogin without-password
 StrictModes yes
 
@@ -73,5 +73,5 @@
 
 # Allow client to pass locale environment variables
-AcceptEnv LANG LC_*
+#AcceptEnv LANG LC_*
 
 Subsystem sftp /usr/lib/openssh/sftp-server
 
