--- /root/.bashrc	2014-09-18 08:17:02.000000000 +0000
+++ /root/.bashrc	2014-11-04 00:00:00.000000000 +0000
@@ -3,5 +3,6 @@
 # Note: PS1 and umask are already set in /etc/profile. You should not
 # need this unless you want different defaults for root.
-# PS1='${debian_chroot:+($debian_chroot)}\h:\w\$ '
+#PS1='\[\e[1;91m\]${debian_chroot:+($debian_chroot)}\u@\h\[\e[0m\]:\[\e[1;94m\]\W\[\e[0m\]\$ '
+PS1='\[\e[1;95m\]\h:\[\e[0m\]\[\e[1;91m\]\W ${debian_chroot:+($debian_chroot)}\u\$\[\e[0m\] '
 # umask 022
 
